### PR TITLE
Test 78 As a user I should be able to click and open the PTO in view only mode

### PR DESCRIPTION
**Pull request by kristiyanplpeev**
Front end - added new page with PTO detailed view.

Back end - added endpoint for getting PTO full info by passing PTO id.

### CODE REVIEW

File: server/src/holidays/holidays.service.ts [hunk 1]
```diff
@@ -285,4 +289,29 @@ export class HolidaysService {
       throw new Error('Something went wrong with getting user holidays');
     }
   }
+
+  private async getPTOFullInfo(PTOId: string): Promise<PTO> {
+    try {
```
Comment (MilenIvanovv) [inline server/src/holidays/holidays.service.ts:294]: There is no point of this try catch

Comment (MilenIvanovv) [reply server/src/holidays/holidays.service.ts:294]: The same message will be thrown without it

File: client/src/inversify/interfaces.ts [hunk 2]
```diff
@@ -13,6 +20,7 @@ export interface HolidaysServiceInterface {
     approvers,
   }: IHolidayFullInfo): Promise<void | { warning: string }>;
   userPTOsRequest(): Promise<UserHolidayType[]>;
```
Comment (MilenIvanovv) [inline client/src/inversify/interfaces.ts:22]: This is not changed in this PR but the Array<> type can be used here.

File: client/src/inversify/interfaces.ts [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+import {
+  IHolidayInfo,
+  HolidayDaysInfoType,
+  UserInfoType,
+  IHolidayFullInfo,
+  UserHolidayType,
+  PTOFullInfo,
+} from "common/types";
 
 export interface UserServiceInterface {
```
Comment (MilenIvanovv) [inline client/src/inversify/interfaces.ts:10]: We use the Interface convention which has "I" at the start in the interface names.

**Review by MilenIvanovv — CHANGES_REQUESTED** [review]
Looks great, i left some comments for minor changes

File: server/src/holidays/holidays.service.ts (hunk 1 above)
Comment (kristiyanplpeev) [reply server/src/holidays/holidays.service.ts:294]: done.

File: client/src/inversify/interfaces.ts (hunk 2 above)
Comment (kristiyanplpeev) [reply client/src/inversify/interfaces.ts:22]: done.

File: client/src/inversify/interfaces.ts (hunk 3 above)
Comment (kristiyanplpeev) [reply client/src/inversify/interfaces.ts:10]: done.

**Review by MilenIvanovv — APPROVED** [review]
LGTM

**Pull request merged**